Update Building in Windows 10 or 11 with MSYS2.md

pwnept opened the pull request:
Changed default git command to checkout with a tag instead of a branch, kept the old command modified as a comment. (because i.e. release_5.0.0 branch does not exist)
Added where to find tags.
Updated version numbers of iNav and ARM toolchain.

pwnept committed:
Added checkout with a tag instead of a branch

Changed default git command to checkout with a tag instead of a branch, but kept the old command as a comment. (i.e. release_5.0.0 branch does not exist)
Added where to find tags (Just in case).
Updated version numbers of iNav and ARM toolchain.

pwnept committed:
Update Building in Windows 10 or 11 with MSYS2.md

DzikuVx commented on docs/development/Building in Windows 10 or 11 with MSYS2.md:
```diff
@@ -54,21 +61,21 @@ This will give you the version you need for any given release or master branch.
-# for version 2.6.1, version needed is 9.2.1
-wget https://github.com/xpack-dev-tools/arm-none-eabi-gcc-xpack/releases/download/v9.2.1-1.1/xpack-arm-none-eabi-gcc-9.2.1-1.1-win32-x64.zip
-unzip xpack-arm-none-eabi-gcc-9.2.1-1.1-win32-x64.zip
+# for iNav version 5.0.0, tookchain version needed is 10.2.1
```
Let's use `INAV` everywhere. All capitals please :)

erstec replied:
Exactly!
And please correct "tookchain"

pwnept replied:
Done!

pwnept committed:
Replaced iNav with INAV and correct typos

pwnept committed:
Replaced targets link to official INAV repo

Replaced targets link from https://github.com/tednv/inav/tree/master/src/main/target to official INAV repo at https://github.com/inavflight/inav/tree/master/src/main/target

pwnept reviewed (commented):
Fixed in [b48adf5](https://github.com/iNavFlight/inav/pull/8248/commits/b48adf56b90d24e302b85b61791648c241771d9a)

pwnept committed:
Merge branch 'iNavFlight:master' into update_building_with_msys2
